pkgs/applications/misc/etesync-dav/default.nix

```
@@ -5,56 +5,7 @@
 , python3
 , fetchPypi
 , radicale3
-}:
-
-let
-  python = python3.override {
-    packageOverrides = self: super: {
-      flask = super.flask.overridePythonAttrs (old: rec {
-        version = "2.0.3";
-        src = old.src.override {
-          inherit version;
-          hash = "sha256-4RIMIoyi9VO0cN9KX6knq2YlhGdSYGmYGz6wqRkCaH0=";
-        };
-
-        patches = [
-          # Pulling in this patch lets us continue running tests without any
-          # other changes using setuptools >= 67.5.0.
-          (fetchpatch {
-            name = "remove-deprecated-pkg-resources.patch";
-            url = "https://github.com/pallets/flask/commit/751d85f3de3f726446bb12e4ddfae885a6645ba1.patch";
-            hash = "sha256-T4vKSSe3P0xtb2/iQjm0RH2Bwk1ZHWiPoX1Ycr63EqU=";
-            includes = [ "src/flask/cli.py" ];
-          })
-        ];
-      });
-      flask-wtf = super.flask-wtf.overridePythonAttrs (old: rec {
-        version = "0.15.1";
-        format = "setuptools";
-        src = old.src.override {
-          inherit version;
-          pname = "Flask-WTF";
-          hash = "sha256-/xdxhfiRMC3CU0N/5jCB56RqTpmsph3+CG+yPlT/8tw=";
-        };
-        disabledTests = [
-          "test_outside_request"
-        ];
-        disabledTestPaths = [
-          "tests/test_form.py"
-          "tests/test_html5.py"
-        ];
-        patches = [ ];
-      });
-      werkzeug = super.werkzeug.overridePythonAttrs (old: rec {
-        version = "2.0.3";
-        src = old.src.override {
-          inherit version;
-          hash = "sha256-uGP4/wV8UiFktgZ8niiwQRYbS+W6TQ2s7qpQoWOCLTw=";
-        };
-      });
-    };
-  };
-in python.pkgs.buildPythonApplication rec {
+}: python3.pkgs.buildPythonApplication rec {
```

Small nit to be more consistent with other expressions. Otherwise LGTM
```suggestion
}: 

python3.pkgs.buildPythonApplication rec {
```